ntbs-service/wwwroot/source/Components/YearComparison.ts

```diff
@@ -0,0 +1,38 @@
+import Vue from 'vue';
+import { getValidationBaseBath } from '../helpers';
+const axios = require('axios');
+
+const YearComparison = Vue.extend({
+    props: ['yeartocompare'],
+    data: function() {

```

Comment:
I should add a comment here that yeartocompare needs to be passed in, and will then be compared to the year in the input field to make clearer how this should be used.